packages/compass-preferences-model/src/preferences.ts

```diff
@@ -278,7 +264,7 @@ const allFeatureFlagsProps: Required<{
   ...featureFlagsProps,
 };
 
-const modelPreferencesProps: Required<{
+const defaultUserPreferencesProps: Required<{
```

Comment:
I'm not quite sure I follow the name here ... :) These are the ones that are stored, right, not defaults?